tango_simlib/sim_xmi_parser.py

```diff
@@ -207,7 +219,11 @@ def parse(self, sim_xmi_file):
         device_class = root.find('classes')
         self.device_class_name = device_class.attrib['name']
         for class_description_data in device_class:
-            if class_description_data.tag in ['commands']:
+            if class_description_data.tag in ['description']:
+                class_description_info = (
+                        self.extract_device_class_descr(class_description_data))
```

Comment:
On github this looks like it is indented a bit too much.  Please check it is only 4 spaces.